code/bit_manipulation/src/addition_using_bits/addition_using_bits.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+#include <string>
```

Comment:
Any reason this is included?

Reply:
Nope, left it there from working on something else. Added fixes, let me know if anything else should be changed.